ibm_catalog.json

```diff
@@ -0,0 +1,193 @@
+                            "key": "create_policy",
+                            "required": true,
+                            "hidden":false   
+                        },
+                        {
+                            "key": "vpn_client_access_group_users",
+                            "required": false,
+                            "hidden":false   
+                        },
+                        {
+                            "key": "access_group_name",
+                            "required": true,
+                            "hidden":false   
+                        }
+                    ],
+                    "iam_permissions": [
+                        {
+                            "role_crns": [
+                                "crn:v1:bluemix:public:iam::::role:Administrator"
+                            ],
+                            "service_name": "iam-identity"
+                        },
+                        {
+                            "role_crns": [
+                                "crn:v1:bluemix:public:iam::::role:Administrator"
+                            ],
+                            "service_name": "is.vpc"
+                        }
+                    ],
+                    "architecture": {
+                        "descriptions": "",
+                        "features": [
+                            {
+                                "title": "Works with all three landing zone variations (VPC, OpenShift and VSI)",
+                                "description": "Yes"
+                            },
+                            {
+                                "title": "Supports OpenVPN Clients",
+                                "description": "Yes"
+                            },
+                            {
+                                "title": "Separate IAM access list for VPN users",
+                                "description": "Yes"
+                            },
+                            {
+                                "title": "Separate VPC subnet for VPN Server",
+                                "description": "Yes"
+                            },
+                            {
+                                "title": "IBM-managed VPN Server",
+                                "description": "Yes"
+                            },
+                            {
+                                "title": "Access to private network via VPN clients",
+                                "description": "Yes"
+                            }
+                        ],
+                        "diagrams": [
+                            {
+                                "diagram": {
+                                    "caption": "Client-to-site VPN configuration for accessing a landing zone.",
+                                    "url": "https://github.com/ccmitchellusa/terraform-ibm-client-to-site-vpn/blob/main/examples/landing-zone/c2s-basic.drawio.svg",
```

Comment:
pointing to a forked repo here

Reply:
Fixed in https://github.com/terraform-ibm-modules/terraform-ibm-client-to-site-vpn/pull/16